pkg/client/gen.go

```diff
@@ -57,9 +58,17 @@ func (c *SonobuoyClient) GenerateManifest(cfg *GenConfig) ([]byte, error) {
 		return nil, errors.Wrap(err, "couldn't marshall selector")
 	}
 
+	// Template values that are regexps (`E2EFocus` and `E2ESkip`) are
```

Comment:
@chuckha Here's an (overly verbose) explanation of why single quotes in YAML strings in this case.